### DISCUSSION TITLE
PayPal not supported

### DISCUSSION THREAD

**Discussion by sanwald7** [woocommerce/woocommerce#51718] [Checkout Flow]
Why is PayPal not supported in the woocommerce block checkout!? It's a plugin developed bei Woo.

**Comment by mikejolley** [comment]
Hi,

PayPal is supported if you install the correct extension. These are the docs for the [newer PayPal extension](https://wordpress.org/plugins/woocommerce-paypal-payments/) showing how it integrates:

https://woocommerce.com/document/woocommerce-paypal-payments/#woocommerce-blocks

You can also enable the legacy PayPal Standard integration which also works with blocks:

https://developer.woocommerce.com/2021/07/12/developer-advisory-paypal-standard-will-be-hidden-on-new-installs/

Hope that helps.
Thanks
